README.md

```diff
@@ -1,10 +1,88 @@
 # CARTA Frontend
+The carta-frontend is used in conjunction with the [carta-backend](https://github.com/CARTAvis/carta-backend), and for general use a pre-built production carta-frontend is usually installed alongside the carta-backend automatically. 
+However, a user may wish to install the carta-frontend component separately in order to test beta versions, for example, or to develop it.
+
+## Pre-built production packages
+The simplest way to install the carta-frontend component is to use the pre-built production packages from NPM (any OS with Node.js installed), our PPA (Debian packages for Ubuntu), our RPM repository (for RHEL based systems), or Homebrew (for MacOS).
+
+### NPM
+
+Pre-built production carta-frontends can be obtained from the [npmjs.com carta-frontend repository](https://www.npmjs.com/package/carta-frontend?activeTab=explore).
+
+If you have Node.js and `npm` already installed on your system, a production carta-frontend can be installed using the following command:
+
+```
+npm i carta-frontend
+```
+The files will be installed to your `node-modules` directory.
```

Comment:
Maybe this is obvious to npm users, but perhaps a brief comment explaining where this directory can be located? (Depending on whether the npm installation is local or system-wide?)